src/features/projects/styles/ProjectSnippet.module.scss

```diff
@@ -117,20 +117,42 @@
   font-family: $primaryFontFamily;
   font-size: $fontMedium;
   color: $light;
-  display: flex;
-  flex-wrap: wrap;
-  column-gap: 5px;
+  padding-right: 16px;
+}
+.verifiedIcon {
+  margin-bottom: -2px;
+  margin-left: 5px;
 }
```

Comment:
While this works, if I were to do somethign like change the font size for the project name, I'd end up needing to make adjustments here as well. It also has the effect of leaving a left margin if the verified icon ends up on a new line for any reason.

Refer this for a possible alternative approach - https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1787/commits/40cd858eba669e966310c09f84985967b2ac3d0d

Reply:
I made minor changes in 271b26be9f7221520a51cd57bb8eba03634b0da4